### PR TITLE
[SAIC-755] CF scenarios with disabled Rollback.

### PR DESCRIPTION
**Pull request by archyufa**
Add CF scenarios for resources and VM migration with NO Rollback.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

File: scenario/stages_without_rollback/1_checkcloud.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# This stage verifies destination cloud is functional by creating
+# tenant, flavor, glance image, private tenant network and subnet,
+# cinder volume as well as spinning VM using API calls.
+# After successful testing all resources and VMs are deleted.
+
+preparation:
+  - create_snapshot: True
+
+process:
+  - act_check_dst_cloud: True
```
Comment (roman-verchikov) [inline scenario/stages_without_rollback/1_checkcloud.yaml]: `act_check_dst_cloud` stage can be done in `preparation` phase

File: scenario/stages_without_rollback/2_identity_transporter.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# NOTE: Rollback has been disabled.
+# Stage to migrate identity resources such as tenants, roles
+# and user tenant roles to destination cloud.
+
+preparation:
+  - create_snapshot: True
+
+process:
+  - act_get_filter: True
+  - act_check_filter: True
```
Comment (roman-verchikov) [inline scenario/stages_without_rollback/2_identity_transporter.yaml]: `act_get_filter` and `act_check_filter` should be in `preparation` stage

File: scenario/stages_without_rollback/3_get_and_transfer_images.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# Stage to migrate glance resources such as images, snapshots and image
+# members.
+
+preparation:
+  - create_snapshot: True
+  - create_image_snapshot_dst: True
+
+process:
+  - act_get_filter: True
+  - act_check_filter: True
```
Comment (roman-verchikov) [inline scenario/stages_without_rollback/3_get_and_transfer_images.yaml]: `act_get_filter` and `act_check_filter` should be in `preparation` stage

File: scenario/stages_without_rollback/4_transport_compute_resources.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# NOTE: Rollback has been disabled.
+# Stage to migrate compute resources such as flavors and nova quotas.
+
+preparation:
+  - create_snapshot: True
+
+process:
+  - act_get_filter: True
+  - act_check_filter: True
```
Comment (roman-verchikov) [inline scenario/stages_without_rollback/4_transport_compute_resources.yaml]: `act_get_filter` and `act_check_filter` should be in `preparation` stage

File: scenario/stages_without_rollback/5_network_transporter.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+preparation:
+  - create_snapshot: True
+
+rollback:
+  - restore_from_snapshot: True
+
+process:
+  - act_get_filter: True
+  - act_check_filter: True
```
Comment (roman-verchikov) [inline scenario/stages_without_rollback/5_network_transporter.yaml]: `act_get_filter` and `act_check_filter` should be in `preparation` stage

File: scenario/stages_without_rollback/6_get_transfer_volumes.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# NOTE: Rollback has been disabled.
+# Stage to migrate cinder resources such as volumes, snapshots, quotas.
+
+preparation:
+  - create_snapshot: True
+
+process:
+  - act_get_filter: True
+  - act_check_filter: True
```
Comment (roman-verchikov) [inline scenario/stages_without_rollback/6_get_transfer_volumes.yaml]: `act_get_filter` and `act_check_filter` should be in `preparation` stage

File: scenario/stages_without_rollback/7_transport_keypairs.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# NOTE: Rollback has been disabled.
+# Stage to migrate keypair resource.
+
+preparation:
+  - create_snapshot: True
+
+process:
+  - act_get_filter: True
+  - act_check_filter: True
```
Comment (roman-verchikov) [inline scenario/stages_without_rollback/7_transport_keypairs.yaml]: `act_get_filter` and `act_check_filter` should be in `preparation` stage

**Comment by roman-verchikov** [comment]
please resolve inlines

File: scenario/stages_without_rollback/5_network_transporter.yaml (hunk 5 above)
Comment (japaniel) [inline scenario/stages_without_rollback/5_network_transporter.yaml]: Are we going to be rolling back to a snapshot for network_transporter?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by MirantisWorkloadMobility** [comment]
Can one of the admins verify this patch?

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**